src/msgpack/MsgPackContext.cs

```diff
@@ -70,6 +70,15 @@ public IMsgPackConverter<T> GetConverter<T>()
                 ?? TryGenerateNullableConverter(type));
         }
 
+        public IMsgPackConverter<T> GetConverter<T>(Type type)
+        {
+            return (IMsgPackConverter<T>)
+                (GetConverterFromCache(type)
+                ?? TryGenerateArrayConverter(type)
```

Comment:
We could remove these methods in next PR, when we will implement TryGenerateGenericConverter.